include/seer.hrl

```diff
@@ -0,0 +1,14 @@
+-define(APP, seer).
+-define(ENV(Key, Default), application:get_env(?APP, Key, Default)).
+-define(ENV_PREFIX, prefix).
+-define(ENV_HOST, host).
+-define(ENV_MODE, mode).
+-define(ENV_INTERVAL, interval).
+-define(ENV_CARBON_HOST, carbon_host).
+-define(ENV_CARBON_PORT, carbon_port).
+-define(DEFAULT_PREFIX, <<"seer">>).
```

Comment:
I have to admit I kinda prefer a function that returns the default config. E.g:

```erlang
defaults() -> [
    {prefix, <<"seer">>},
    {host, <<"localhost">>},
    {mode, carbon},
    {carbon_host, localhost},
    {carbon_port, 2003},
    {interval, 10000}
].
```

This affords doing some clean lists merging with `defaults()` and `application:get_all_env(seer)`

That said, it's just my opinion. I also haven't thought very much about the merits of these macros; I always suspected it was cargo-culting.